src/Divi/Repositories/Forms.php

```diff
@@ -36,6 +36,28 @@ public function getAll()
         return $forms;
     }
 
+    /**
+     * @unreleased
+     *
+     * @param array $forms
+     *
+     * @return array
+     */
+    public function getV3Forms($forms)
+    {
+        $output = [];
+
+        foreach ($forms as $id) {
+            $isV3Form = (bool)give()->form_meta->get_meta($id, 'formBuilderSettings', true);
+
```

Comment:
Let's use the `Utils::isV3Form` here as well

Reply:
Resolved https://github.com/impress-org/give-divi/pull/46/commits/9f2cec2920145a71256e44144ccc8615390b247c